portal/src/test/scala/cromwell/pipeline/utils/auth/TestUserUtils.scala

```diff
@@ -0,0 +1,23 @@
+package cromwell.pipeline.utils.auth
+
+import java.util.UUID
+
+import cromwell.pipeline.datastorage.dto.{ User, UserId }
+import cromwell.pipeline.utils.StringUtils
+
+object TestUserUtils {
+  def getDummyUser(password: String = "-Pa$$w0rd-", passwordSalt: String = "salt", active: Boolean = true): User = {
```

Comment:
As far as `password` could be used in other tests, it would be better to create `val` for it

Reply:
Done